src/Prelude.mo

```diff
@@ -15,19 +15,19 @@ module {
   /// Each have calls are well-typed in all typing contexts, which
   /// trap in all execution contexts.
   public func nyi() : None {
-    assert false; loop { }
+    Debug.trap("Prelude.nyi()");
   };
 
   public func xxx() : None {
```

Comment:
What is the purpose of this function? Can we remove it?

Reply:
I assumed it is for a hole, but all of these functions (and module) should be deprecated then deleted, in my opinion.